NEWS.md

```diff
@@ -1,3 +1,22 @@
+# Updated contribution workflow. 
+
+The team of main-developers of t8code and contributors to t8code is getting bigger and we needed an improved workflow to manage all of our contributions.
+With the latest version of t8code we decided to use github project boards to have a good track of our development. Having a look at [t8code's Issue Landing page](https://github.com/orgs/DLR-AMR/projects/13) you will find the state of all issues concerning t8code. If you have a good solution for an Issue that has the state "ToDo" / is in the "ToDo"-column, feel free to assign yourself and work on it. It is highly appreciated. In the future we will have specialized Project-boards, where Issues to a certain topic are summarized. But at first, all Issues will occur on t8code's Issue Landing page.
```

Comment:
```suggestion
With the latest version of t8code, we decided to use github project boards to have a good track of our development. Having a look at [t8code's Issue Landing page](https://github.com/orgs/DLR-AMR/projects/13), you will find the state of all issues concerning t8code. If you have a good solution for an issue that has the state "ToDo" / is in the "ToDo"-column, feel free to assign yourself and work on it. It is highly appreciated. In the future we will have specialized Project-boards, where issues to a certain topic are summarized. But at first, all issues will occur on t8code's Issue Landing page.
```

Reply:
Maybe a matter of style but i would use more commas. And is there a reason that you capitalized Issue sometimes?